### PR TITLE
Webedit prs: cargo packs edition

### PR DESCRIPTION
**Pull request by spessbro**
## About The Pull Request
More cargo packs and goodies obviously all subject to change and suggestions are welcome :)
-3.5k clarke crate
-1250 cr. bluespace shelter goodie
-chemical crate has all the base chem dispenser chems and now in the same order
-2k advanced rped crate with 5 of each tier 2 stock part

## Why It's Good For The Game
More things for assistants who gamed the bounty system to buy to make the wealth circulate more.

## Changelog
:cl:
add: Added new crates and goodie see about
add: stable plasma bottle for the chem crate
tweak: tweaked price of chemical starter kit and added the rest of the chem dispenser reagents
/:cl:

### CODE REVIEW

File: code/modules/cargo/packs.dm [hunk 1]
```diff
@@ -672,6 +672,23 @@
 					/obj/item/mecha_parts/mecha_equipment/hydraulic_clamp)
 	crate_name= "APLU MK-I kit"
 
+/datum/supply_pack/engineering/clarke
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:675]: not an engineering mech

File: code/modules/cargo/packs.dm [hunk 2]
```diff
@@ -672,6 +672,23 @@
...
+	name = "APLU MK-I Crate"
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:676]: wrong name

File: code/modules/cargo/packs.dm [hunk 3]
```diff
@@ -672,6 +672,23 @@
...
+					/obj/item/mecha_parts/part/clarke_torso,
+					/obj/item/mecha_parts/part/clarke_right_arm,
+					/obj/item/mecha_parts/part/clarke_left_arm,
+					/obj/item/stock_parts/capacitor,
+					/obj/item/stock_parts/scanning_module,
+					/obj/item/circuitboard/mecha/clarke/main,
+					/obj/item/circuitboard/mecha/clarke/peripherals,
+					/obj/item/mecha_parts/mecha_equipment/drill,
+					/obj/item/mecha_parts/mecha_equipment/hydraulic_clamp,
+					/obj/item/mecha_parts/mecha_equipment/orebox_manager)
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:689]: have you looked at the code for this

File: code/modules/cargo/packs.dm [hunk 4]
```diff
@@ -672,6 +672,23 @@
...
+	contains = list(/obj/item/mecha_parts/chassis/clarke,
+					/obj/item/mecha_parts/part/clarke_torso,
+					/obj/item/mecha_parts/part/clarke_right_arm,
+					/obj/item/mecha_parts/part/clarke_left_arm,
+					/obj/item/stock_parts/capacitor,
+					/obj/item/stock_parts/scanning_module,
+					/obj/item/circuitboard/mecha/clarke/main,
+					/obj/item/circuitboard/mecha/clarke/peripherals,
+					/obj/item/mecha_parts/mecha_equipment/drill,
+					/obj/item/mecha_parts/mecha_equipment/hydraulic_clamp,
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:688]: clarke doesnt operate a clamp

File: code/modules/cargo/packs.dm [hunk 5]
```diff
@@ -1396,6 +1426,13 @@
 	contains = list(/obj/item/storage/part_replacer/cargo)
 	crate_name = "\improper RPED crate"
 
+/datum/supply_pack/science/rped/tier2
+	name = "Advanced RPED crate"
+	desc = "Upgraded parts for upgraded price. Contains less parts than the regular one."
+	cost = 2000
+	contains = list(/obj/item/storage/part_replacer/cargo/advanced)
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:1433]: wrong typepath

**Comment by Fikou** [comment]
verified/10

File: code/modules/cargo/packs.dm [hunk 6]
```diff
@@ -1577,6 +1597,23 @@
...
+	contains = list(/obj/item/mecha_parts/chassis/clarke,
+					/obj/item/mecha_parts/part/clarke_torso,
+					/obj/item/mecha_parts/part/clarke_right_arm,
+					/obj/item/mecha_parts/part/clarke_left_arm,
+					/obj/item/stock_parts/capacitor,
+					/obj/item/stock_parts/scanning_module,
+					/obj/item/circuitboard/mecha/clarke/main,
+					/obj/item/circuitboard/mecha/clarke/peripherals,
+					/obj/item/mecha_parts/mecha_equipment/drill,
+					/obj/item/mecha_parts/mecha_equipment/orebox_manager)
```
Comment (Fikou) [inline code/modules/cargo/packs.dm:1614]: remove this

Comment (spessbro) [reply code/modules/cargo/packs.dm:1614]: why

**Comment by optimumtact** [comment]
webedit ☑️ 
balance change ☑️ 
cringe ☑️ 

we have the trifecta

**Comment by optimumtact** [comment]
I think my favourite is the crate that lets you just ignore science ☑️

**Comment by MrDoomBringer** [comment]
smh
can we get a smh tag

**Comment by spessbro** [comment]
i cri everytim

**Comment by kriskog** [comment]
lol

File: code/modules/cargo/packs.dm (hunk 6 above)
Comment (Fikou) [reply code/modules/cargo/packs.dm:1614]: read the code

**Comment by ExcessiveUseOfCobblestone** [comment]
does it work?

**Comment by tralezab** [comment]
He didn't change the orebox one, so nope

**Comment by ExcessiveUseOfCobblestone** [comment]
If you are contributing a multiple-file change do not do it through the web editing feature of github.

**Pull request closed without merging**